asf_core_data/pipeline/preprocessing/feature_engineering.py

```diff
@@ -353,9 +356,11 @@ def get_heating_features(df, fine_grained_HP_types=False):
 def get_postcode_coordinates(df):
     """Add coordinates (longitude and latitude) to the dataframe
     based on the postcode.
+    Postcode fields must be named POSTCODE.
 
     Args:
         df (pandas.DataFrame): EPC dataframe.
+        postcode_field_name: Field name for postcode. Defaults to 'POSTCODE'.
```

Comment:
this is not currently an argument

Reply:
Yep, fixed. (Weird, I thought I'd already commited exactly this fix. )